src/core/ddsi/src/q_radmin.c

```diff
@@ -14,6 +14,7 @@
 #include <stdarg.h>
 #include <stdlib.h>
 #include <limits.h>
+#include <stdalign.h>
```

Comment:
were you going to use this? It seems to be C++ and later.

Reply:
@dodsonmg no surprise there, it is very hard to spot the error on Windows, but this here is indeed the reason:
```
C:\Users\travis\build\eclipse-cyclonedds\cyclonedds\src\core\ddsi\src\q_radmin.c(17): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory [C:\Users\travis\build\eclipse-cyclonedds\cyclonedds\build\src\core\ddsc.vcxproj]
```

Reply:
I agree.  This is unnecessary since I'm using `sizeof` instead of `alignof`.